packages/design-system/src/components/Button/Button.js

```diff
@@ -69,10 +70,15 @@ const Button = React.forwardRef((props, ref) => {
       aria-expanded={ariaExpanded}
     >
       {loading && (
-        <Loader size="small" className={styles[`${baseClass}__loader`]} />
+        <Loader
+          size="small"
+          label={loaderLabel}
+          className={styles[`${baseClass}__loader`]}
+          labelClassName={styles[`${baseClass}__loader-label`]}
+        />
       )}
       {icon && <i className={styles[`${baseClass}__icon`]}>{icon}</i>}
-      {children}
+      {children && <div>{children}</div>}
```

Comment:
Why do we change it? It may break some implementations, tests, or custom stylings. I'm not saying it's bad, butI would like to know why do we change it :-)

Reply:
Actually, regardless of reason Sambor had to change it, we need that ;D take a look a the thread in Agent App: https://github.com/livechat/my.livechatinc.com/pull/12849#discussion_r406655529
google translate doesn't like our buttons with so-called "loose text" when loading or icon is rendered ;x

Reply:
The reason behind that change is that the "loose text" is incompatible with [that line](https://github.com/livechat/design-system/pull/179/files#diff-ce3c4a2a8c708ff0a84602f45be811a5R116).

Reply:
cool beans 👌